authentication/index.adoc

```diff
@@ -56,6 +56,6 @@ You can manage authorization for {product-title} through the following tasks:
 ====
 After creating the cluster admin user, xref:../authentication/remove-kubeadmin.adoc#removing-kubeadmin_removing-kubeadmin[delete the existing kubeadmin user] to improve cluster security.
 ====
-* Creating service accounts: xref:../authentication/understanding-and-creating-service-accounts.adoc#service-accounts-overview_understanding-service-accounts[Service accounts] provide a flexible way to control API access without sharing a regular user’s credentials. A user can xref:../authentication/understanding-and-creating-service-accounts.adoc#service-accounts-managing_understanding-service-accounts[create] and xref:../authentication/using-service-accounts-in-applications.adoc#service-accounts-using-credentials-externally_using-service-accounts[use a service account in applications] and also as xref:../authentication/using-service-accounts-as-oauth-client.adoc#using-service-accounts-as-oauth-client[an OAuth client].
+* Creating service accounts: xref:../authentication/understanding-and-creating-service-accounts.adoc#service-accounts-overview_understanding-service-accounts[Service accounts] provide a flexible way to control API access without sharing a regular user’s credentials. A user can xref:../authentication/understanding-and-creating-service-accounts.adoc#service-accounts-managing_understanding-service-accounts[create and use a service account in applications] and also as xref:../authentication/using-service-accounts-as-oauth-client.adoc#using-service-accounts-as-oauth-client[an OAuth client].
```

Comment:
> [create and use a service account in applications]

`create and use a service account in applications` includes 2 parts: one is `create`, the other is `use a service account in applications`. Your this change seems to give `create and use a service account in applications` with a hyperlink to only the `create` stuff: xref:../authentication/understanding-and-creating-service-accounts.adoc#service-accounts-managing_understanding-service-accounts, however would lack the `use a service account in applications` stuff?

Reply:
@xingxingxia I saw that also. Thank you for confirming that I need to make that change.

Reply:
> I need to make that change.

@mburke5678 , saw this PR is merged without making that change, do you mean you'll make that change in separate PR?